config/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     database_url: str = "postgresql+asyncpg://authdb_owner:V97gEkaPchLf@ep-plain-water-a5scbycn.us-east-2.aws.neon.tech/authdb"
-    secret_key: str = "e705c2325058744e7328d842be8da79a1f7c934363b1f061c7934d4804f3d360"
+    secret_key: str = "empty_string_1"
     algorithm: str = "HS256"
```

Comment:
Hardcoding the `secret_key` as 'empty_string_1' is insecure. It's crucial to use a strong, randomly generated secret key for JWT authentication to ensure the security of the tokens. Consider generating a secure key and storing it securely, for example, in environment variables.

**Label**: security

**Code suggestion**

```python
secret_key: str = os.getenv("SECRET_KEY")
```